### ISSUE TITLE
Ignoring the commit after a commit message body

### ISSUE THREAD

**Issue by WesleyGoncalves** [fojuth/readmegen#29]
In case you have 3 commits and the first commit message has a body, for example:

```sh
0c0260f feat: Add feature A

        The feature A is responsible for doing A.
d671128 fix: Fix the issue #12
f921bbd docs: Add documentation
```

the **second commit** message(`fix`) won't appear in the CHANGELOG file because the parser does not separate the **description of the second commit message** (`fix:...`) from the **body of the first commit**("The feature A..."). 

So the output in this case will be

```md
#### Features
* Add feature A

#### Documentation
* Add documentation
```

when it should be 

```md
#### Features
* Add feature A

#### Bugfixes
* Fix the issue #12

#### Documentation
* Add documentation
```

**Comment by WesleyGoncalves** [comment]
I found out that the `git log` command is outputting the **body message** and I believe it is not necessary because, to keep a CHANGELOG file clean, it must have less text and the **description** of the commit message should be sufficient.
Therefore, I removed the `%b` placeholder so the body of the git commit message will be ignored. In other words, only the **description**/**subject** of the git commit message will be considered.

**Comment by WesleyGoncalves** [comment]
It may also fix the issue #4

**Comment by WesleyGoncalves** [comment]
The commit [`86e33ab` ](https://github.com/fojuth/readmegen/commit/86e33ab531b906680377db77b3feb15d65e05d02) fix this issue.
